### PR TITLE
fix(mariadb): Show the mysql config directly

### PR DESCRIPTION
**Pull request by NSSPKrishna**
# Summary

As part of Shape Up week, we at Virtuoso are aiming to enhance the user experience for MariaDB integration. To achieve this, we are implementing a streamlined process that allows users to directly access the Install flow page, bypassing the intermediary suggestion page.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by NSSPKrishna** [comment]
I have made the necessary changes to the branch. Please review @pkosanam

**Comment by pkosanam** [comment]
Hi @NSSPKrishna , as discussed, mariadb datasource is not hosted. Hence you are seeing the below error while running install framework repository in your local.
Datasource list: https://onenr.io/0kjnp9eB6wo

image

File: data-sources/mariadb/config.yml [hunk 1]
```diff
@@ -6,14 +6,17 @@ description: |
 icon: logo.svg
 install:
   primary:
+    nerdlet:
```
Comment (pkosanam) [inline data-sources/mariadb/config.yml:9]: can remove these changes as your marketplace changes are not hosted on master branch.

your marketplace PR: https://source.datanerd.us/nr1-dev-experience/marketplace/pull/968

**Comment by pkosanam** [comment]
Also, I have explained you that, when a community datasource(datasource which is hosted on quickstart repository) is displayed on "Add data" page, then we see "Begin installation" button.

Here is an example.
1. community datasource: langchain
image
2. On clicking on it, we see "Begin installation" button
image
3. Only after clicking on it, we will see the installation flow, which was created on marketplace repository
image

Hope this description helps.

File: data-sources/mariadb/config.yml (hunk 1 above)
Comment (NSSPKrishna) [reply data-sources/mariadb/config.yml:9]: Sure making the changes

Comment (NSSPKrishna) [reply data-sources/mariadb/config.yml:9]: will get the marketplace pr merged and raise another pr for this changes

**Review by MichelLosier — APPROVED** [review]

**Pull request merged**